src/app/pages/ledger/connect-ledger-error/connect-ledger-error.layout.tsx

```diff
@@ -0,0 +1,75 @@
+import { FiCircle } from 'react-icons/fi';
+import { Box, color, Flex, Stack } from '@stacks/ui';
+
+import { BaseDrawer } from '@app/components/drawer';
+import { Caption } from '@app/components/typography';
+import ConnectLedgerError from '@assets/images/onboarding/ledger/connect-ledger-error.png';
+import { ErrorLabel } from '@app/components/error-label';
+import { PrimaryButton } from '@app/components/primary-button';
+import { Link } from '@app/components/link';
+
+import { ConnectLedgerTitle } from '../components/connect-ledger-title';
+
+interface PossibleReasonUnableToConnectProps {
+  text: string;
+}
+function PossibleReasonUnableToConnect(props: PossibleReasonUnableToConnectProps) {
+  const { text } = props;
+
+  return (
+    <Flex>
+      <Box mr="tight" mt="3px">
```

Comment:
The margin top is to visually align the icon, right? this is a good trick, although I'd check how it scales when the font is larger.

Reply:
I am not sure what you mean here, scaling the font in the browser or in code? Do you have another suggestion?

Reply:
Think he means when you <kbd>cmd</kbd> + <kbd>+</kbd>

Reply:
I do, please ignore this comment. It's almost impossible to fix it.